packages/server/src/sdk/app/rows/search/external.ts

```diff
@@ -69,24 +75,22 @@ export async function search(
   }
 
   try {
-    let rows = await handleRequest(Operation.READ, tableId, {
+    const parameters = {
       filters: query,
       sort,
       paginate: paginateObj as PaginationJson,
       includeSqlRelationships: IncludeRelationship.INCLUDE,
-    })
+    }
+    let rows = await handleRequest(Operation.READ, tableId, parameters)
+    let totalRows: number | undefined
+    if (countRows) {
+      totalRows = await handleRequest(Operation.COUNT, tableId, parameters)
+    }
```

Comment:
Is it worth structuring these in such a way that you can issue both of the `handleRequest` calls and wrap them in a `Promise.all` so that they're executing at the same time?